MIGRATION.md

```diff
@@ -0,0 +1,46 @@
+<h1>Migration Guide</h1>
+
+- [From version 1.x to 2.0.x](#from-version-1x-to-20x)
+  - [Date Input](#date-input)
+
+# From version 1.x to 2.0.x
+
+## Date Input
+
+We updated the React  DatePicker from 1.X to 3.X. The library no longer uses moment. The library uses DateFns as dependency for the date management. Therefore, The Date Input no longer uses moment object for the value, but a Javascript Date object. Also, the onChange Callbackk return a Date value instead of a moment Value.
```

Comment:
The library uses -> Instead it uses

Reply:
Callbackk > callback

Reply:
DateFns as dependency for the date management > DateFns as a dependency for date management